tests/kernel/install_ltp.pm

```diff
@@ -224,12 +221,27 @@ sub add_ltp_repo {
     zypper_ar($repo, name => 'ltp_repo');
 }
 
+sub get_default_pkg {
+    if (is_sle) {
+        return 'qa_test_ltp' if (is_released);
```

Comment:
The default for released SLES should be `qa_test_ltp qa_test_ltp-32bit`. I'll probably add 32bit syscalls and cves to the schedule soon.

Reply:
Sure, I'll fix it.